src/Services/TokenService.php

```diff
@@ -122,7 +122,9 @@ public function obtainTokens()
     public function actAs(int $id)
     {
         $response = $this->pendingRequest()
-            ->asForm()->post('oauth/token', [
+            ->asForm()->withToken(
+                $this->token()
+            )->post('oauth/token', [
```

Comment:
This was the part which was failing during recording, all other tests were almost fine